docusaurus/docs/dev-docs/cli.md

```diff
@@ -3,5 +3,413 @@ title: Command Line Interface
+
+This command is to run a Strapi application without restarts and file writes, primarily for use in production.
+Certain features such as the Content-type Builder are disabled in the `strapi start` mode because they require application restarts. The `start` command can be prefaced with [environment variables](/dev-docs/configurations/environment.md#strapi-s-environment-variables) to customize the application start.
+
+## strapi build
+
+Builds your admin panel.
+
+```bash
+strapi build
+
+options: [--no-optimization]
+```
+
+- **strapi build**<br/>
+  Builds the administration panel and delete the previous build and .cache folders
+- **strapi build --no-optimization**<br/>
+  Builds the administration panel without minimizing the assets. The build duration is faster.
+
+## strapi watch-admin
+
+Starts the admin server. Strapi should already be running with `strapi develop`.
+
+```sh
+strapi watch-admin
+options: [--browser <name>]
+```
+
+## strapi export <BetaBadge />
+
+Exports your project data. The default settings create a `.tar` file, compressed using `gzip` and encrypted using `aes-128-ecb`.
+
+```console
+strapi export
+```
+
+The exported file is automatically named using the format `export_YYYYMMDDHHMMSS` with the current date and timestamp. Alternately, you can specify the filename using the `-f` or `--file` flag. The following table provides all of the available options as command line flags:
+
+| Option           | Type    | Description                                                                                               |
+|------------------|---------|----------------------------------------------------------------------------|
+| --no-encrypt     |         | Disables file encryption and disables the `key` option.                                                   |
+| --no-compress    |         | Disables file compression.                                                                                |
+| --key            | string  | Passes the encryption key as part of the `export` command. The `--key` option can't be combined with `--no-encrypt`. |
+| --max-size-jsonl | integer | Splits the internal `jsonl` files when exceeding the specified size in MB (default 256MB).                                |
+| -f, --file       | file    | Specifies the export filename. Do not include a file extension.                                           |
+| -h, --help       |         | Displays help for the `strapi export` command.                                                            |
+
+<!--| --exclude        | string  | Comma-separated list of data to exclude (files [localMediaFiles, providerMediaFiles], content [entities, links], schema, configuration). |         |  example: strapi export --exclude XXXXXX #exports your data excluding XXXXX.-->
+
+**Examples**
+
+```sh
+#examples of strapi export:
+
+strapi export -f myData #exports your data with the default options and the filename myData.
```

Comment:
I'm wondering which "language" identifier we could use because it seems like the rendered block does not seem to support syntax coloring (comments do not appear as such). Any idea? 🤔 `console` maybe? I'll try to find the full list of supported languages somewhere.

Reply:
I could also move the comment to a title, right?

Reply:
You could move the first comment, yes ("#examples of strapi export").

Reply:
It seems like `bash` works. The strange thing is it colors `export` in the .md file but it looks fine in the HTML.